lib/style.scss

```diff
@@ -2,6 +2,15 @@
 @use '@carbon/react/scss/zone';
 @use '@carbon/react/scss/reset';
 
+:root {
```

Comment:
Very dangerous, you leak to the global namespace, without proper namespacing.

Consider options to fix:

* Give that variable a prefixed (namespaced) name, i.e. `--bio-vo-focus...`
* Define the variable on the root scope of the variable outline component

Reply:
Removed, since we are not handling carbon theme in this issue.